src/websocket/types/room/objects/extension.rs

```diff
@@ -12,9 +13,10 @@ with_structure_fields_and_update_struct! {
         #[serde(default, rename = "off")]
         pub disabled: bool,
         /// The current amount of energy held in this structure.
```

Comment:
The comments here & in spawn.rs are now out of date. Could probably just use a find/replace for `energy` -> `resources`?